cockpit-frontend/src/components/TeamMemberList/TeamMemberList.jsx

```diff
@@ -186,16 +196,35 @@ export default function TeamMemberList() {
         {mvpInfo.team && (
           <Grid item xs={6}>
             {!update && (
-              <Button
-                className={classes.addButton}
-                variant="outlined"
-                color="primary"
-                startIcon={<AddIcon />}
-                onClick={displayForm}
-              >
-                Add A Member
-              </Button>
+              <Grid container spacing={1}>
```

Comment:
In this script line 172, 2 lines are commented, is it necessary to keep them ?

{/*  <span className={classes.emailText}>{member.email}</span>
                      <br /> */}

Reply:
done